models/channel.go

```diff
@@ -3,12 +3,19 @@ package models
 import (
```

Comment:
Template logic should be served from handlers. Not in models.

Model only deal with data. Presentation layer should be extracted out of it